students/PhongT/lesson02/series.py

```diff
@@ -0,0 +1,94 @@
+"""
+Lession 02
+Fibonacci Series Exercise
+"""
+
+
+def fibonacci(n):
+    """The Fibonacci Series"""
+    if n <= 0:
+        print("Invalid input: n must be >= 0")
+    elif n == 1:
+        return 0
+    elif n == 2:
+        return 1
+    else:
+        return fibonacci(n-1) + fibonacci(n-2)
+
+
+def lucas(n):
+    """The Lucas Number"""
+    if n <= 0:
+        print("Invalid input: n must be >= 0")
+    elif n == 1:
+        return 2
+    elif n == 2:
+        return 1
+    else:
+        return lucas(n-1) + lucas(n-2)
+
+
+
+def sum_series(n, f0 = 0, f1 = 1):
```

Comment:
try to use full words as variable names whenever possible.  f0 and f1 are not descriptive even though i know what you mean.